spec/fixtures/katello-certs-check/ca.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Why do we need to store the private key and CSR? They're quite useless for our purposes, right? If we do need to update, we can just generate a new one.

Reply:
We do not. Figured they were not hurting anything to have the all the pieces present.

Reply:
I generally don't like having unused files in a repo. I've also had bots trigger on private keys in a repo and received emails from that which me lean to not including the files. Perhaps a README on how it was generated would be nice?